views/partials/sign-up-banners/monitor-features.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="row flx-col jst-cntr {{ variableClasses }}">
     <div class="bg-white gradient-inset">
       <div class="col-12 flx-cntr txt-cntr feat-headline-wrapper">
-        <h2 class="headline feature-tip-headline">
+        <h2 data-lang="" class="headline feature-tip-headline">
```

Comment:
Do we still need this since it looks like we're applying this to all locales? I don't see any reference to `dataset.lang` in `public` (anymore?)

Reply:
Nope- I'll remove!